client/src/components/Designer/Designer.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import './Designer.css'
+
+class Designer extends React.Component {
```

Comment:
Does this component have state? If not, can you change it into a functional component?